CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ The present file will list all changes made to the project; according to the
 
 ### Deprecated
 - Usage of XML-RPC API is deprecated.
+- `Netpoint` replaced by `Socket` and table `glpi_netpoints` renamed to `glpi_sockets`
```

Comment:
Please move this in `Deprecated` section of `API changes`

Reply:
done